### ISSUE TITLE
Could id3 correctly parse tag generated by audacity on CJK system?

### ISSUE THREAD

**Issue by kalxd** [polyfloyd/rust-id3#91]
Hi, all:

I modified one audio in the [audacity](https://www.audacityteam.org/), also change the metadata by using audacity editor and saved as `abc.mp3`:

![metadata editor](https://fars.ee/cRZ4.png)

here is the plain text:
* title: 王小明独唱
* album: 王小明专辑
* artist: 王小明

however, I used `id3` to parse the tag:

```rust
let tag = Tag::read_from_path("abc.mp3");
dbg!(tag);
```
it said 

```
[src/main.rs:78] Tag::read_from_path(path) = Err(
    StringDecoding([]): data is not valid utf-8,
)
```
So is there a way to parse tag correctly? or it is the issue of audacity?

thanks.

**Comment by polyfloyd** [comment]
It should be able to. Could you please attach the problematic file?

**Comment by kalxd** [comment]
> It should be able to. Could you please attach the problematic file?

of course, here is two files: `origin.mp3` is the origin audio, and `abc.mp3` is the product of audacity.

the link: [https://github.com/kalxd/kalxd.github.io/tree/master/attachment](https://github.com/kalxd/kalxd.github.io/tree/master/attachment)

`id3` can parse the `origin.mp3`, but not the `abc.mp3`, I didn't know the reason either.

**Comment by polyfloyd** [comment]
Thanks! Will investigate when I have time

**Comment by polyfloyd** [comment]
Fixed! Thank you for reporting :)
